src/main/java/com/bootcamp/demo/firebase/FirebaseController.java

```diff
@@ -1,25 +1,16 @@
-package com.bootcamp.demo;
+package com.bootcamp.demo.firebase;
```

Comment:
Would it be better to have all controller in the same package, to keep the layering?